src/app/components/quotes/quotes.hbs

```diff
@@ -1,36 +1,36 @@
 <style scoped>
-    @import url(/app/pages/home/quotes/quotes.css);
+    @import url(/app/components/quotes/quotes.css);
 </style>
 <div class="quotes">
     <div class="quote-bucket right">
-        <img src="/images/home/quotes/juan.jpg" width="175" height="230" />
+        <div class="image juan"></div>
```

Comment:
I'm just talking about the name of the class for the div. Having it "juan" ties it to specific content, when it should be about function or structure. Not a real stopper, just something to consider.